designer/server/src/views/400.njk

```diff
@@ -0,0 +1,11 @@
+{% extends 'layouts/page.njk' %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
```

Comment:
Slowly trying to switch to double quotes everywhere in Nunjucks

```suggestion
{% extends "layouts/page.njk" %}

{% from "govuk/components/button/macro.njk" import govukButton %}
```

Reply:
Done for a few pages I found